public/pages/Snapshots/components/SnapshotRestoreAdvancedOptions/SnapshotRestoreAdvancedOptions.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiCheckbox, EuiSpacer, EuiText } from "@elastic/eui";
+import CustomLabel from "../../../../components/CustomLabel";
+import React, { ChangeEvent } from "react";
+
+interface SnapshotAdvancedOptionsProps {
+  restoreAliases: boolean;
+  onRestoreAliasesToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  restoreClusterState: boolean;
+  onRestoreClusterStateToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  ignoreUnavailable: boolean;
+  onIgnoreUnavailableToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  restorePartial: boolean;
+  onRestorePartialToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  customizeIndexSettings: boolean;
+  onCustomizeIndexSettingsToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  ignoreIndexSettings: boolean;
+  onIgnoreIndexSettingsToggle: (e: ChangeEvent<HTMLInputElement>) => void;
+  width?: string;
+}
+
+const SnapshotRestoreAdvancedOptions = ({
+  restoreAliases,
+  onRestoreAliasesToggle,
+  ignoreUnavailable,
+  onIgnoreUnavailableToggle,
+  restoreClusterState,
+  onRestoreClusterStateToggle,
+  restorePartial,
+  onRestorePartialToggle,
+  customizeIndexSettings,
+  onCustomizeIndexSettingsToggle,
+  ignoreIndexSettings,
+  onIgnoreIndexSettingsToggle,
+  width,
+}: SnapshotAdvancedOptionsProps) => (
+  <div style={{ padding: "10px 10px", width: width }}>
+    <EuiCheckbox
+      id="restore_aliases"
+      label={<CustomLabel title="Restore aliases" helpText="Restore index aliases alongside their associated indices" />}
```

Comment:
Minor detail, but I believe the convention is for `helpText` and error messages to end with a hard stop (`.`).

Reply:
fixed and pushed